js/forceAtlas.js

```diff
@@ -18,58 +21,66 @@ export function drawForceAtlas(edgeList, nodeList, colorValues) {
     
     const width = +svg.attr('width') + 1400 - margin.left;
     const height = +svg.attr('height') + 1000 - margin.top;
-
-    // var color = d3.scaleOrdinal(d3.schemeCategory20);
+    
     var color = d3.scaleOrdinal()
         .domain(colorValues)
-        .range(["#f7fbff", "#e3eef9", "#cfe1f2", "#b5d4e9", "#93c3df", "#6daed5", "#4b97c9", "#2f7ebc", "#1864aa", "#0a4a90", "#08306b"]);
-
-    var centralitySize = d3.scaleLinear()
-    	.domain([d3.min(nodeList, function(d) { return d.degree; }), d3.max(nodeList, function(d) { return d.degree; })])
-	.range([15,50]);
-
-    var edgeWidth = d3.scaleLinear()
```

Comment:
Everything here works great and is straightforward. I'm curious why you got rid of the D3 scales and what you're using in their places. I see `d.scaled_weight` below, but I'm not sure where that's coming from or when it's introduced.

Reply:
I'm also seeing the `radius_$` for node size, which seems to be doing the same work as the scale. Sorry if I'm missing something obvious!

Reply:
Ya I moved these into main.js because I figured we would be using them across all three graphs potentially. So instead of calling it multiple times, I just added these properties directly to the nodes and edges. Happy to switch back if you think it makes more sense to have this code be specific for each graph!

Reply:
Great! I was just looking in the wrong place. This change makes a lot of sense, much more convenient/organized.

Reply:
Tbh I'm pretty much always guilty of starting to reorganize code and then getting distracted 😅... I already see spots where I added variables I'm not using or places I should be using changes that I didn't 🤷🏽‍♀️ so definitely appreciate your eyes 🙏🏽